src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarRegionSwitcher.tsx

```diff
@@ -31,19 +35,25 @@ export const BrowserNavBarRegionSwitcher = (props: Props) => {
       <div className={styles.regionFieldWrapper}>
         <BrowserRegionField />
       </div>
-      <div className={styles.regionEditorWrapper}>
-        <BrowserRegionEditor />
-      </div>
+      {props.breakpointWidth >= BreakpointWidth.BIG_DESKTOP && (
+        <div className={styles.regionEditorWrapper}>
+          <BrowserRegionEditor />
+        </div>
+      )}
     </div>
   );
 };
 
+const mapStateToProps = (state: RootState) => ({
```

Comment:
The component's immediate parent can tell it whether or not to show the second navigation component.

Reply:
This clashes with the previous comment. If we are going to send the `breakpointWidth` via the parent, then should we still change the prop name to `display`?

Reply:
Sure:

`<BrowserNavBarRegionSwitcher display={breakpoint satisfies condition ? 'full' : 'compact'} />`

Reply:
For this to be cleaner, won't we need an enum instead of using `full` and `compact` as strings?

Reply:
✅